### PR TITLE
scan: Implement Subscription and wire up for Delegation on account index page

### LINKED ISSUE #1136: Implement Subscription and wire up for Delegation on account index page
Display delegation on account index page for each account

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1136

### CODE REVIEW

File: scan/src/subscriptions/DelegationSub.re [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        SingleConfig.makeVariables(
+          ~delegator_address=delegator_address |> Address.toBech32,
+          ~validator_address=validator_address |> Address.toOperatorBech32,
+          (),
+        ),
+    );
+  result |> Sub.map(_, x => x##delegations_by_pk);
+};
+
+let getList = delegator_address => {
```
Comment (evilpeach) [inline scan/src/subscriptions/DelegationSub.re]: Don't use snake case.

File: scan/src/subscriptions/DelegationSub.re [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  let (result, _) =
+    ApolloHooks.useSubscription(
+      TotalStakeConfig.definition,
+      ~variables=
+        TotalStakeConfig.makeVariables(
+          ~delegator_address=delegator_address |> Address.toBech32,
+          (),
+        ),
+    );
+  Js.Console.log(result);
```
Comment (evilpeach) [inline scan/src/subscriptions/DelegationSub.re]: remove log

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/components/account/AccountIndexDelegations.re [hunk 3]
```diff
@@ -9,12 +9,12 @@ module Styles = {
 };
 
 [@react.component]
-let make = (~delegations: list(AccountHook.Account.delegation_t)) => {
+let make = (~delegatorStake: Js.Array.t(DelegationSub.stake_t)) => {
```
Comment (evilpeach) [inline scan/src/components/account/AccountIndexDelegations.re]: Let's use `array(..)` instead of `Js.Array.t`

**Comment by tansawit** [comment]
Re-uploaded at PR #1180

**Pull request closed without merging**